docs/icd.md

```diff
@@ -8,59 +8,88 @@
 
 ## Overview
 
-This document details the common software interfaces required by all services in the Arrow ecosystem.
+This document details the endpoints provided by the `svc-pricing` service.
+
+Currently, the server is configured to be live on `localhost:50051`, and is subject to change when deployed to production. This address can be interfaced by any API platforms like Postman, or even `curl`. 
+
+However, `svc-pricing` is intended to be consumed by other Arrow micro-services through the `svc-pricing-client` library, rather than the server being directly connected to and interacted with frontend clients.
```

Comment:
A bit confusing. You could state that autogenerated client-side functions for this gRPC server are conveniently packaged in the `svc-pricing-client` crate, to be used to ask for pricing by svc-cargo, svc-rideshare, and other customer-facing services.

Reply:
[5559de2](https://github.com/Arrow-air/svc-pricing/pull/2/commits/5559de2b620e5c50c257f63d87648c96d6e7b3ca)